tests/ssp/tests/01-run.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
```

Comment:
no (C) aabadie? (:

Reply:
btw, is pep8 not telling you to remove the shebang here also?

Reply:
> no (C) aabadie? (:

copy-pasta, there might be others in _pexpect_ PRs

> is pep8 not telling you to remove the shebang here also?

pep8 is not telling anything about shebang. And we want to keep the shebang here, since this script is directly executed by make (see [here](https://github.com/RIOT-OS/RIOT/pull/8004/files#diff-15fb6046811bc0ad1d4dca2da3c29163R17))